developer-docs-site/src/css/custom.css

```diff
@@ -1243,6 +1243,10 @@ html[data-theme='dark'] .sidebar-divider {
   width: 80%;
 }
 
+.move-content a[name] {
+  scroll-margin-top: calc(var(--ifm-navbar-height) + 0.5rem);
+}
+
```

Comment:
this is an American hero... thank you, thank you... I spent hours on this problems :facepalm:

Reply:
🇺🇸🇺🇸🇺🇸